moreinterp/go.mod

```diff
@@ -1,3 +1,16 @@
 module mvdan.cc/sh/moreinterp
 
-go 1.23
+go 1.24.0
+
+toolchain go1.24.4
```

Comment:
rather surprising and a bit worrying that u-root already requires Go 1.24 or later; as a project we support two Go versions, so ideally I'd like to keep the same for the coreutils module. Perhaps we can ask them to lower this.

at the very least I think we can remove the toolchain line?

Reply:
Opened https://github.com/u-root/u-root/pull/3428 with the proposal.

In the meantime, I removed the toolchain line.